src/Http/Controllers/CountryController.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Konekt\AppShell\Http\Controllers;
+
+use Illuminate\Contracts\View\View;
+use Illuminate\Http\RedirectResponse;
+use Konekt\Address\Models\Country;
```

Comment:
We need to type hint the interface so that Laravel resolves the right model
```suggestion
use Konekt\Address\Contracts\Country;
```